code4me-server/markdowns/index.md

```diff
@@ -1,25 +1,28 @@
 # Code4Me
-**Code4Me** is a plugin that autocompletes code in a TU Delft research project for the CSE3000 course given in the Computer Science and Engineering bachelor.
-This study focuses on the usefulness of various statement completion language models in a real-world setting.
+Code4Me provides language model based code completion. Code4Me predicts statement (line) completion and is available for both Jetbrains and Visual Studio Code. The code suggestions from Code4Me can be recognised by the logo in the suggestion menu. Code4Me triggers (on specific trigger characters) automatically or it can be prompted manually by pressing the keybind. This keybind differs per IDE.
```

Comment:
```suggestion
Code4Me provides language model based code completion. Code4Me predicts statement (line) completion and is available for both PyCharm (but also other JetBrains IDEs) and Visual Studio Code. The code suggestions from Code4Me can be recognised by the logo in the suggestion menu. Code4Me triggers (on specific trigger characters) automatically or it can be prompted manually by pressing the keybind. This keybind differs per IDE: **alt + shift + K** for the JetBrains plugin, and **ctrl + space** for the VSC plugin.
```

Reply:
> 

```suggestion
Code4Me provides automatic intelligent code completion based on large pre-trained language models. Code4Me predicts statement (line) completion and is available for both PyCharm (also other JetBrains IDEs) and Visual Studio Code. The code suggestions from Code4Me can be recognised by the logo in the suggestion menu. Code4Me automatically triggers (on specific trigger characters) or the user can prompt it manually by pressing the keybind. The keybind differs per IDE: **alt + shift + K** for the JetBrains plugin, and **ctrl + space** for the VSC plugin.```